src/Differ.php

```diff
@@ -40,6 +45,14 @@ public function __construct($header = "--- Original\n+++ New\n")
     }
 
     /**
+     * @param bool $show
+     */
+    public function setShowNonDiffLines($show)
```

Comment:
I thought about that, but wasn't sure, as someone may want to turn this off but keep the heading. However, now that I think about it, if you're using this option, you probably don't want the heading either, and so are likely to just pass in an empty string as the first parameter. I'll adjust the setting to go into the constructor.